kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/resolver/IngressResolutionResult.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator.resolver;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxy;
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxyIngress;
+import io.kroxylicious.kubernetes.api.v1alpha1.virtualkafkaclusterspec.Ingresses;
+import io.kroxylicious.kubernetes.operator.model.ResolvedIngress;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+
+/**
+ * The result of dereferencing an Ingresses ingressRef and proxyRef
+ * @param ingress the ingress
+ * @param ingressResolutionResult the dereferenced KafkaProxyIngress result
+ * @param proxyResolutionResult the dereferenced KafkaProxy result
+ */
+public record IngressResolutionResult(Ingresses ingress,
```

Comment:
I don't know what to make of the plural type but singular parameter name.

Reply:
Again, I don't really understand why the need for a `Result` thing and a `Resolved` thing.

Reply:
`Ingresses` is generated code from the CRD, I think this record-field is unused.

Reply:
have removed the `Resolved` classes